src/shared/input-wrapper/input-wrapper.tsx

```diff
@@ -36,9 +57,9 @@ export const InputWrapper = styled.div<InputWrapperStyleProps>`
         props.$position === "right" ? "row-reverse" : "row"};
 
     :focus-within {
-        border: 1px solid ${Color.Accent.Light[1]};
-        box-shadow: ${DesignToken.InputBoxShadow};
+        ${defaultFocusCss}
     }
+    ${(props) => props.$focused && defaultFocusCss}
 
```

Comment:
need to handle focus programmatically, as the floating element is not necessarily a child in the actual HTML DOM hierarchy